app/javascript/tests/enrollment/steps/ExposureInformation.test.js

```diff
@@ -47,9 +47,9 @@ describe('ExposureInformation', () => {
     expect(wrapper.find(Form).exists()).toBe(true);
     expect(wrapper.find('#exposure_notes').exists()).toBe(true);
     expect(wrapper.find(PublicHealthManagement).exists()).toBe(true);
-    expect(wrapper.find(Button).length).toEqual(2);
-    expect(wrapper.find(Button).at(0).text()).toEqual('Previous');
-    expect(wrapper.find(Button).at(1).text()).toEqual('Next');
+    expect(wrapper.find(Button).length).toEqual(3);
+    expect(wrapper.find(Button).at(1).text()).toEqual('Previous');
+    expect(wrapper.find(Button).at(2).text()).toEqual('Next');
```

Comment:
this is now more confusing since they're not the only buttons on the page.  I'd throw classes on them and just use that

Reply:
hold on this comment I'm fixing with my test updates